File: peek_plugin_unified_search/_private/storage/ExcludedKwTuple.py
from sqlalchemy import Column
from sqlalchemy import Integer, String
from vortex.Tuple import Tuple, addTupleType

from peek_plugin_unified_search._private.PluginNames import unifiedSearchTuplePrefix
from peek_plugin_unified_search._private.storage.DeclarativeBase import DeclarativeBase


@addTupleType
class ExcludedKwTuple(Tuple, DeclarativeBase):
    __tupleType__ = unifiedSearchTuplePrefix + 'ExcludedKwTuple'
    __tablename__ = 'ExcludedKwTuple'

    id = Column(Integer, primary_key=True, autoincrement=True)
    string1 = Column(String(50))
    int1 = Column(Integer)

File: peek_plugin_unified_search/plugin-module/_private/tuples/ExcludedKwTuple.ts
import {addTupleType, Tuple} from "@synerty/vortexjs";
import {unifiedSearchTuplePrefix} from "../PluginNames";


@addTupleType
export class ExcludedKwTuple extends Tuple {
    public static readonly tupleName = unifiedSearchTuplePrefix + "ExcludedKwTuple";

    //  Description of date1
    id : number;

    //  Description of string1
    string1 : string;

    //  Description of int1
    int1 : number;

    constructor() {
        super(ExcludedKwTuple.tupleName)
    }
}